pkgs/development/python-modules/spox/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, python3Packages
+, fetchPypi
+ }:
```

```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, numpy
, onnx
, onnxruntime
, packaging
, pytestCheckHook
, setuptools-scm
```

we cannot use python3Packages here because that draws in python 3.11